FFXCutsceneRemover/Components/BrotherhoodTransition.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FFX_Cutscene_Remover.ComponentUtil;
+using System;
+
+namespace FFXCutsceneRemover
+{
+    /* Special transition for the 'Yuna says goodbye to Besaid' cutscene. Many things get added, enabled and equipped here.
+       Would have been a crazy amount of MemoryWatchers to add, most of which are only used once for this cutscene, so just
+       figured it made more sense to have one special transition here that derives from the Transition class */
+    class BrotherhoodTransition : Transition
```

Comment:
As commented here, there is a insane amount of things we need to enable after this cutscene is skipped. 

Rather than add them all to the memory watcher where most of them will never be used again, I opted to have a child of the Transition class which I added to the Dictionary so the common addresses can be written to without cluttering the MemoryWatcher with single use addresses.

I quite like this design and think we can tidy up a lot of other skips where there are single use addresses and keep the parent Transition class for the common addresses.